### DISCUSSION TITLE
HTTPS for the transmission rpc web interface?

### DISCUSSION THREAD

**Discussion by dorianps** [haugene/docker-transmission-openvpn#2631] [Q&A]
Does anyone know if there is an easy way to serve the transmission web gui RPC over https instead of regular http. I find it weird that the RPC password I set can probably be sniffed in the home network by anyone with minimal knowledge.

**Comment by pkishino** [comment]
please see discussions on the official transmission repo on WHY no effort is put into https support.. if you still want it, setup a e.g. nginx reverse proxy
